bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/adapters/ColumnDescriptorAdapter.java

```diff
@@ -194,8 +196,9 @@ public static String buildGarbageCollectionExpression(HColumnDescriptor columnDe
   private static void convertGarbageCollectionExpression(String gcExpression,
       HColumnDescriptor columnDescriptor) {
     if (Strings.isNullOrEmpty(gcExpression)) {
-      // No GC Expression means keep all versions, MaxVersions = 0.
-      columnDescriptor.setMaxVersions(0);
+      // No GC Expression means keep all versions. HBase doesn't support this, but this is the
```

Comment:
nit: No GC expression in Bigtable means keeping all versions. HBase doesn't support keeping all version, so we translate it by setting the max version to Integer.MAX_VALUE.